test/helper/auth0.js

```diff
@@ -0,0 +1,34 @@
+const { before } = require('mocha')
```

Comment:
could use `nock` and `mock-jwks` to avoid using real auth endpoint for tests? Like in `vitalam-api` https://github.com/digicatapult/vitalam-api/blob/main/test/integration/routes.test.js

Reply:
yes this is a good idea. I'll do this